src/lib/coda_lib/coda_lib.ml

```diff
@@ -792,7 +797,9 @@ module Make (Inputs : Inputs_intf) = struct
                  (External_transition.of_verified
                     (With_hash.data transition_with_hash)) )) ;
         don't_wait_for
-          (Linear_pipe.transfer_id (Net.states net) external_transitions_writer) ;
+          (Strict_pipe.transfer
+             (Strict_pipe.Reader.of_linear_pipe (Net.states net))
```

Comment:
can you change the `Net`'s use of Linear pipe to Strict pipe as well?

Reply:
Sure, I can have a look on that.